internal/pkg/term/selector/local_selector.go

```diff
@@ -219,6 +281,44 @@ func (s *localFileSelector) listDockerfiles() ([]string, error) {
 	return dockerfiles, nil
 }
 
+// listDirsAndFiles returns the list of directories and files within the current
+// working directory and two subdirectory levels below.
+func (s *localFileSelector) listDirsAndFiles() ([]string, error) {
+	names, err := s.getDirAndFileNames(s.ws.ProjectRoot(), 0)
```

Comment:
i wonder if the `depth` passed here should be 3 and we instead subtract in `getDirAndFileNames` until depth == 0 - that way the _user_ of the `getDirAndFileNames` function decides how deep to search!

Reply:
Done.